config/standards/PMD/ruleset.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ruleset name="DVSA-V2">

```

Comment:
This was taken as a first draft template of rules based on a previous project for DVSA (Driving and Vehicle Standards Agency). I figured it would be a good basis for rules since they are based on Government standards. Its also an oversight on my part. Good spot. I'll change this to OPG.